### ISSUE TITLE
AR1.1.4B Mining Rocket Ship Duplication

### ISSUE THREAD

**Issue by shank3001** [zmaster587/AdvancedRocketry#549]
Every time I restart my game on AR 1.1.4B my mining rocket ship is duplicated and stack on top on the prior one. I tried created a few different pads and they are all duplicating. See screenshot below:

<img width="1280" alt="screen shot 2017-04-11 at 6 35 00 am" src="https://cloud.githubusercontent.com/assets/10300791/24911825/25593686-1e81-11e7-8744-f4c17fcd7cfe.png">

**Comment by zmaster587** [comment]
Should be fixed once and for all in 1.1.5.  After the update you may have
it dupe one more time but after that it should stop.  The issue was caused
by the rockets not being removed from the save so when the server started
it thought there were several mission that just finishes

**Comment by shank3001** [comment]
Updated to AR 1.1.5. Checking now. . .

**Comment by zmaster587** [comment]
Can you get back with the results when you get a chance so I can know if there's still a problem?

**Comment by shank3001** [comment]
Sorry, It took me a while to get the data to launch another mining mission. I launched a mining mission under 1.1.14b and it was not pretty as it caused a server tick and corrupted my world. I launched a backup and upgraded to 1.1.15 and it duplicated the mining ship on the first world load, but was good after that. I launched a mining mission without an issue. I did take me several attempts to get the observatory to show the second tab (breaking and replacing different blocks).
